util/fipstools/acvp/acvptool/subprocess/aead.go

```diff
@@ -86,6 +88,19 @@ func (a *aead) Process(vectorSet []byte, m Transactable) (interface{}, error) {
 			return nil, fmt.Errorf("test group %d has unknown direction %q", group.ID, group.Direction)
 		}
 
+		// We automatically assume the IV is given, if the IV generation method is not defined.
+		var external_iv bool = true
+		if group.IvGen != "" {
+		    switch group.IvGen {
+		    case "external":
+		        external_iv = true
+		    case "internal":
+		        external_iv = false
+		    default:
+		        return nil, fmt.Errorf("test group %d has unknown iv generation method %q", group.ID, group.IvGen)
+		    }
+		}
```

Comment:
Is the comment `CryptoAlg-814?selectedConversation=f8040bf0-a270-44f6-9684-2a9199e0ad68` addressed?
The default treatment is not applicable to below algorithms because the data does not provide 'ivGen' filed.

ACVP-AES-CCM
ACVP-AES-KW
ACVP-AES-KWP

Reply:
Yes, the commit 6019b353f49bf8999eb9a9c1bed24e8e9afec419 included in this PR was a fix for that.